### PR TITLE
21.9.0 release notes

### PR DESCRIPTION
**Pull request by bryce-anderson**

### CODE REVIEW

File: source/blog/2021-09-30-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+September release, hot off the press:
+
+[Util](https://github.com/twitter/util/)
+========================================
+
+New Features
+------------
+
+-   util-jvm: Experimentally crossbuilds with Scala 3. [bfdde071](https://github.com/twitter/util/commit/bfdde0713ea93f29f2fe72800313c0b66f3d9e0b)
+-   util-stats: Counter, Gauge, and Stat can be instrumented with descriptions. `PHAB_ID = D615481`
```
Comment (enbnt) [inline source/blog/2021-09-30-release-notes.md:20]: PHAB_ID wasn't replaced

File: source/blog/2021-09-30-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+-   util-sl4j-api: Experimentally crossbuilds with Scala 3. [e2368780](https://github.com/twitter/util/commit/e23687809145b8366046deaa8a7639e97c1152d6)
+-   util-sl4j-jul-bridge: Experimentally crossbuilds with Scala 3. [16a4a945](https://github.com/twitter/util/commit/16a4a945d1107a3030944595babb250e010954c8)
+-   util-stats: Experimentally crossbuilds with Scala 3. [ea54f59c](https://github.com/twitter/util/commit/ea54f59c0d31b2cd37065af59dcd1d37a39e6a09)
+-   util-zk-test: Experimentally crossbuilds with Scala 3. [86df882b](https://github.com/twitter/util/commit/86df882b076c2c8df8eef3bd2e5e467a2089486b)
+-   util-app: Flags parsing will now roll-up multiple flag parsing errors into a single
+    error message. When an error is encountered, flag parsing will continue to collect parse error
+    information instead of escaping on the first flag failure. After parsing all flags, if any errors
+    are present, a message containing all of the failed flags and their error reason,
+    along with the help usage message will be emitted. \`0def519d &lt;https://github.com/twitter/util/commit/0def519daf1cfd5dbed03faba750fbf331672a4e&gt;\_\_
```
Comment (enbnt) [inline source/blog/2021-09-30-release-notes.md:32]: Formatting didn't convert correctly, needs fixing

File: source/blog/2021-09-30-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+--------------------
+
+-   util-app: the c.t.app.App\#flags field is now a final def instead of a val to address
+    override val scenarios where the c.t.app.App\#flags are accessed as part of construction,
+    resulting in a NullPointerException due to access ordering issues.
+    The c.t.app.Flags class is now made final. The c.t.app.App\#name field has changed from
+    a val and is now a def. A new c.t.app.App\#includeGlobalFlags def has been exposed, which
+    defaults to true. The c.t.app\#includeGlobalFlags def can be overridden to false
+    (ex: override protected def includeGlobalFlags: Boolean = false) in order to skip discovery
+    of GlobalFlags during flag parsing. \`b44e820c &lt;https://github.com/twitter/util/commit/b44e820c7992746a24fc98e509c749035ded288f&gt;\_\_
```
Comment (enbnt) [inline source/blog/2021-09-30-release-notes.md:50]: Formatting didn't convert correctly, needs fixing

File: source/blog/2021-09-30-release-notes.md (hunk 1 above)
Comment (yufangong) [inline source/blog/2021-09-30-release-notes.md:20]: seems the script missed this phab to commit

File: source/blog/2021-09-30-release-notes.md (hunk 2 above)
Comment (yufangong) [inline source/blog/2021-09-30-release-notes.md:32]: some hiccups here too.

File: source/blog/2021-09-30-release-notes.md (hunk 3 above)
Comment (yufangong) [inline source/blog/2021-09-30-release-notes.md:50]: ditto

File: source/blog/2021-09-30-release-notes.md (hunk 1 above)
Comment (bryce-anderson) [reply source/blog/2021-09-30-release-notes.md:20]: Looks like someone added spaces between the `=`.

**Review by yufangong — APPROVED** [review]

**Review by enbnt — APPROVED** [review]

**Pull request merged**